userguide/aviate/aviate-usage-tutorial.adoc

```diff
@@ -0,0 +1,277 @@
+= Usage Tutorial - AI Use Case
+
+include::{sourcedir}/aviate/includes/aviate-card.adoc[]
+
+== Introduction
+
+This tutorial shows how to use the Aviate APIs together with open-source Kill Bill APIs to address usage-based billing scenarios.
+
+Kill Bill already supports sophisticated catalogs, usage-based plans, and automated invoicing. Aviate extends these capabilities with:
+
+* A scalable metering system for capturing usage data points.
```

Comment:
We should update the link from https://github.com/killbill/killbill-docs/blob/f8dcb502fc963bdca102daaa2721924d5c7adf69/userguide/tutorials/consumable_in_arrear.adoc?plain=1#L124 to point to Aviate docs instead.